webapp/src/main.ts

```diff
@@ -2,9 +2,17 @@ import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 import { NestExpressApplication } from "@nestjs/platform-express";
 import { join } from "path";
-import { registerPartials } from "./partials";
 import * as helpers from "handlebars-helpers";
 import { handlebars } from "hbs";
+import { registerPartials } from "./partials";
+
+handlebars.registerHelper("isdefined", function (value) {
+  return value !== undefined;
+});
+
+handlebars.registerHelper("isundefined", function (value) {
+  return value == undefined;
+});
```

Comment:
I prefer camelCase for better readability.
```suggestion
handlebars.registerHelper("isDefined", function (value) {
  return value !== undefined;
});

handlebars.registerHelper("isUndefined", function (value) {
  return value == undefined;
});
```